.gitignore

```diff
@@ -14,4 +16,33 @@ node_modules
 *.tsbuildinfo
 .history
 
+**/*/build
+
+# Logs
+logs
```

Comment:
Does this have an effect on https://github.com/expo/expo-cli/tree/f88f72faa5845e12827d24534c8e171be97e92b7/packages/xdl/src/logs ? And does some package actually use this path to store logs?

Reply:
it doesn't but I made the path a bit more specific anyways since it seems like it could cause issues in the future.